### PR TITLE
docs: Minor Updates to Getting Started

### PR DESCRIPTION
**Pull request by codydaig**
Fixes #969

### CODE REVIEW

File: _includes/docs/0.4.x/gettingstarted.html [hunk 1]
```diff
@@ -39,7 +39,7 @@
     There are several ways you can get the MEAN.JS boilerplate:
   </p>
   <h3>Yo Generator</h3>
-  <p>The generator is still in development. See the <a href="https://github.com/meanjs/generator-meanjs">Generator Repo</a> for information on how to use the development version.
+  <p>Although the main generator is released, the sub generators are still under development. See the <a href="https://github.com/meanjs/generator-meanjs">Generator Repo</a> for information on how to use main generator.
```
Comment (lirantal) [inline _includes/docs/0.4.x/gettingstarted.html]: Do we want to refer to it as the "main generator" or is there another name we can use to make more sense? just maybe "meanjs generator" or something.

at the end of the line it reads "how to use main generator" probably should have a "the" there.

**Comment by lirantal** [comment]
Other than the comments LGTM

**Comment by codydaig** [comment]
@lirantal I updated.

Side Note: Welcome Back! Good to see your face in the notifications again.

**Comment by lirantal** [comment]
LGTM.

Thanks :)
I'm still very awfully busy but doing my best to catch up. You guys are doing a great job as always.

**Pull request merged**